atomisticparsers/utils/mdanalysis.py

```diff
@@ -70,6 +70,24 @@ def universe(self):
                 self.logger.error('Error creating MDAnalysis universe.', exc_info=e)
         return self._file_handler
 
+    @property
```

Comment:
I think property is not appropriate here. Normally you use property when you have a quantity that you use multiple times should not be recalculated. Maybe just name it `calculate_bead_groups`?

Reply:
I made it a property because it is something that (so far) is needed in calculate_molecular_rdf() and calculate_molecular_mean_squared_displacement(). You may access neither, one, or both of these. So, I wanted to ensure that you only calculated the bead_groups once if necessary.